### PR TITLE
Add CODEOWNERS for /content/plugin/ files

### PR DESCRIPTION
**Pull request by bflad**
Reference: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners

Until this documentation is potentially migrated into the terraform-plugin-* repositories, flag the Terraform Provider Development Experience team for code reviews of `/content/plugin/` changes.

### CODE REVIEW

**Review by detro — APPROVED** [review]
I can approve but it's a "gray tick": I don't have permissions on this repo.

**Comment by laurapacilio** [comment]
@bflad do you need additional approval to merge? Just poking since this has been open for a while :)

File: .github/CODEOWNERS
Comment (bflad) [inline .github/CODEOWNERS:2]: Team ID was updated this morning.

```suggestion
/content/plugin/ @hashicorp/terraform-devex
```

**Comment by bflad** [comment]
@laurapacilio we need a current administrator of the repository to ensure `@hashicorp/terraform-devex` has write access in the Settings > Collaborators tab, then they can merge this.

**Comment by laurapacilio** [comment]
I'll talk to Web Dev about this for you :). Also - @bflad can you rebase against `master`?  I think that's why the preview is failing on this one.

**Comment by bflad** [comment]
Rebased 👍

**Pull request merged**
